src/audits/colors.js

```diff
@@ -36,31 +36,29 @@ module.exports = function ( files = [] ) {
 		),
 	];
 
-	return [
-		{
-			id: 'unique',
-			label: 'Number of unique colors',
-			value: uniqColors.length,
-		},
-		{
-			id: 'unique-opaque',
-			label: 'Number of unique colors (ignoring opacity)',
-			value: uniqOpaqueColors.length,
-		},
-		{
-			id: 'all-colors',
-			label: 'List of all colors',
-			value: uniqColors,
-		},
-		{
-			id: 'top-10-colors',
-			label: 'Top 10 most-used colors',
-			value: colorsByCount.slice( 0, 10 ),
-		},
-		{
-			id: 'bottom-10-colors',
-			label: 'Top 10 least-used colors',
-			value: colorsByCount.slice( -10 ).reverse(),
-		},
-	];
+	return {
+		audit: 'colors',
+		data: [
```

Comment:
This change is breaking the default CLI result:

```
yarn css-audit ../css/style.css --colors              

undefined:

undefined
```

I know the grouping is needed for the HTML sections, but it could be done with a group ID in each result object? Then the HTML formatter can group them by that key. Like…

```js
// in audits/colors.js
{
	id: 'top-10-colors',
	audit: 'colors',
	label: 'Top 10 most-used colors',
	value: colorsByCount.slice( 0, 10 ),
},
// in formats/report.js
const colorReports = reports.filter( ( { audit } ) => 'colors' === audit );
```

Reply:
Ah whoops 🤦🏻‍♀️ - yes, this can work! Thanks.

Reply:
@ryelle I updated the PR to use this structure, and the CLI is working. We can send each audit separately to the templates to support different partials that can have varied markup, and there is a `default.twig` partial that can be used as a catch all where needed.